tesseract/tesseract_scene_graph/src/graph.cpp

```diff
@@ -303,6 +303,8 @@ bool SceneGraph::isCollisionAllowed(const std::string& link_name1, const std::st
 
 AllowedCollisionMatrix::ConstPtr SceneGraph::getAllowedCollisionMatrix() const { return acm_; }
 
+AllowedCollisionMatrix::Ptr SceneGraph::getAllowedCollisionMatrix() { return acm_; }
```

Comment:
I believe it is best to keep the implementation separate from the declaration.

Reply:
I'm happy doing whatever you think is best here, but is that from the cpp core guidelines? I just ask because doesn't putting it in the header allow it to be inlined? Probably not super important here, but there were some other files in this PR that had a lot of getters.